### ISSUE TITLE
System.ArgumentException: Negative percentage not allowed

### ISSUE THREAD

**Issue by ElyonsNIC** [EPPlusSoftware/EPPlus#184]
After upgrading from 4.5.3.1 to 5.2.1 we get an exception System.ArgumentException: Negative percentage not allowed when adding multiple series to a chart.  After debugging the source code, the issue is with how the color manager is trying to figure out the color for each series.  The test case (AddColumnChartSingleSerieWithSecondSerieWithCategory) works as expected because it is using the Cycle color method, but we are using the WithinLinear.  The function GetLinearVariation divides by zero when the series is greater than 0 and the numberOfitems equals 1.  The following is the modified test case that reproduces the error.

Thanks
```` 
        [TestMethod]
        public void AddColumnChartSingleSerieWithSecondSerieWithCategory()
        {
            var ws = _pck.Workbook.Worksheets.Add("Column");
            LoadHierarkiTestData(ws);
            var chart = ws.Drawings.AddBarChart("Bar1", eBarChartType.Column3D);

            //Change chart colorMethod from Cylce to WithinLinear
            chart.StyleManager.SetChartStyle(OfficeOpenXml.Drawing.Chart.Style.ePresetChartStyleMultiSeries.Column3dChartStyle1,
                OfficeOpenXml.Drawing.Chart.Style.ePresetChartColors.ColorfulPalette1);
            chart.StyleManager.ColorsManager.Method = OfficeOpenXml.Drawing.Chart.Style.eChartColorStyleMethod.WithinLinear;

            //make series only have range of 1 so that the serie2(index=1) is the same as the number of cells in the range
            //which causes System.ArgumentException: Negative percentage not allowed
            var serie1 = chart.Series.Add(ws.Cells["D2"]);
            var serie2 = chart.Series.Add(ws.Cells["D2"], ws.Cells["C2"]);
            
            //var serie1 = chart.Series.Add(ws.Cells["D2:D17"]);
            //var serie2 = chart.Series.Add(ws.Cells["D2:D17"], ws.Cells["C2:C17"]);
````

**Comment by JanKallman** [comment]
Thanks for reporting it. I'll provide a fix shortly.

**Comment by JanKallman** [comment]
Fixed in version 5.3.2
